src/computation/logic/algorithm-utilities.ts

```diff
@@ -138,12 +167,30 @@ export function getDenominator(algorithm: AlgorithmType, numberOfSeatsAssigned:
             }
         case AlgorithmType.D_HONDT:
             return numberOfSeatsAssigned + 1;
+        case AlgorithmType.LARGEST_FRACTION_HARE:
+        case AlgorithmType.LARGEST_FRACTION_DROOP:
+        case AlgorithmType.LARGEST_FRACTION_HAGENBACH_BISCHOFF:
+            return getElectionNumber(algorithm, totalVotes, totalSeats);
         default:
             console.error(`ERROR! ${algorithm.toString()} does not have an associated denominator function!`);
```

Comment:
This is missing the fancy function that checks the type for all cases being checked.

Reply:
On it.

Reply:
Did this get resolved? I don't see it in the default case.

Reply:
Yes, but the fix did not touch the lines 170-173, it happened in 174+.